### PR TITLE
Restart mac client after update

### PR DESCRIPTION
**Pull request by TheOneRing**

### CODE REVIEW

**Comment by TheOneRing** [comment]
See https://github.com/owncloud/enterprise/issues/3922

**Comment by guruz** [comment]
Since macosx.pkgproj.cmake is a beast, could you please document in the oC handbook how you edited it :-)
Last time I tried some years ago the tools were messy

**Comment by TheOneRing** [comment]
Well it is terribly messy.
I generated a pkgproj, opened it with the tool, and manually reapplied our cmake parameters....

**Comment by TheOneRing** [comment]
@michaelstingl could you give https://jenkins.int.owncloud.com/job/client-osx/13435/ a try

File: admin/osx/post_install.sh.cmake [hunk 1]
```diff
@@ -1,7 +1,9 @@
-#!/bin/sh
+#!/bin/bash
```
Comment (dschmidt) [inline admin/osx/post_install.sh.cmake:1]: Even better usually is /usr/bin/env bash, but probably doesnt matter here.

File: admin/osx/post_install.sh.cmake [hunk 2]
```diff
@@ -10,4 +12,10 @@ if [ -x "$(command -v pluginkit)" ]; then
     pluginkit -e use -i @APPLICATION_REV_DOMAIN@.FinderSyncExt
 fi
 
+if [[ -f ${INSTALLER_TEMP}/OC_RESTART_NEEDED ]]; then
```
Comment (dschmidt) [inline admin/osx/post_install.sh.cmake:15]: quotes

**Pull request merged**